Workbooks/LogAnalytics Workspace/AtScale/Workspace Insights At Scale.workbook

```diff
@@ -0,0 +1,455 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "{Subscription}"
+        ],
+        "parameters": [
+          {
+            "id": "1f74ed9a-e3ed-498d-bd5b-f68f3836a117",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscription",
+            "type": 6,
+            "description": "All subscriptions",
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "where type =~ 'microsoft.operationalinsights/workspaces'\r\n| distinct subscriptionId\r\n//| summarize Count = count() by subscriptionId\r\n//| order by Count desc\r\n//| extend Rank = row_number()\r\n//| project value = subscriptionId, label = subscriptionId, selected = Rank == 1",
```

Comment:
When a user opens this template, they will see no data:
![image](https://user-images.githubusercontent.com/7664931/102917217-a65efe00-4439-11eb-9f1e-cb4e0a891b6b.png)

Can you change this query to:
```
where type =~ 'microsoft.operationalinsights/workspaces'
| summarize Count = count() by subscriptionId
| order by Count desc
| extend Rank = row_number()
| project value = subscriptionId, label = subscriptionId, selected = Rank == 1
```
So that by default one of the subscriptions will be selected?

Reply:
Thanks, changed the query